### PR TITLE
Hardware release 2.0.5

### PR DESCRIPTION
**Pull request by Kampi**

### CODE REVIEW

**Comment by jakkra** [comment]
Checked some dev-kits and they always provide VBUS so should be fine.
image

**Comment by jakkra** [comment]
Is this needed? This is not a power input per se, just reference voltage for IO.
image

**Comment by Kampi** [comment]
> Is this needed? This is not a power input per se, just reference voltage for IO. image

The [reference circuit](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fstruct_pmic%2Fstruct%2Fnpm1300.html) is using a 100 nF cap on this pin.

**Comment by Kampi** [comment]
> > Is this needed? This is not a power input per se, just reference voltage for IO.
> 
> The [reference circuit](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fstruct_pmic%2Fstruct%2Fnpm1300.html) is using a 100 nF cap on this pin.

**Comment by jakkra** [comment]
> > Is this needed? This is not a power input per se, just reference voltage for IO.
> 
> The [reference circuit](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fstruct_pmic%2Fstruct%2Fnpm1300.html) is using a 100 nF cap on this pin.

I looked at this one, which didn't. 


Screenshot_2024-01-30-08-39-44-85_40deb401b9ffe8e1df2f1cc5ba480b12

**Comment by Kampi** [comment]
> > > Is this needed? This is not a power input per se, just reference voltage for IO.
> > 
> > 
> > The [reference circuit](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fstruct_pmic%2Fstruct%2Fnpm1300.html) is using a 100 nF cap on this pin.
> 
> I looked at this one, which didn't.
> 

This doesn´t look like a reference circuit to me. Please check `Configuration 1` under `Hardware and layout` -> `Reference circuitry`

image

**Review by jakkra — APPROVED** [review]
Remove silkscreen for buttons
image

**Comment by Kampi** [comment]
> Remove silkscreen for buttons

Why? I leave it to identify the buttons :)

**Comment by jakkra** [comment]
> > Remove silkscreen for buttons
> 
> Why? I leave it to identify the buttons :)

Ah haha. A bit ugly, could we do just like BTN1-4 or SW1-4?

**Comment by Kampi** [comment]
> > > Remove silkscreen for buttons
> > 
> > 
> > Why? I leave it to identify the buttons :)
> 
> Ah haha. A bit ugly, could we do just like BTN1-4 or SW1-4?

Sure. I will change it

**Comment by Kampi** [comment]
Had to rename them "SWx", because it´s the shortest solution and I don´t have that much space :D

image

**Comment by jakkra** [comment]
> Had to rename them "SWx", because it´s the shortest solution and I don´t have that much space :D
> 
> image

Perfect!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # ZSWatch Hardware
 
+[![License](https://img.shields.io/badge/License-GPL%203.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
```
Comment (jakkra) [inline README.md:3]: Badge is GPL, but the URL links to Apache => update URL to https://opensource.org/license/gpl-3-0/
Same on the dock

Comment (Kampi) [reply README.md:3]: @jakkra it´s the same link as in the ZSWatch Repo README. So we have to change it there too

**Comment by jakkra** [comment]
Are the image in README supposted to work? `docs/images/Preview.png` is missing. Or are you waiting until final done to add it?

**Comment by Kampi** [comment]
> Are the image in README supposted to work? `docs/images/Preview.png` is missing. Or are you waiting until final done to add it?

The link should work, but is dead currently. The best scenario would be to add a link to the CI/CD pipeline and the image there, but I haven´t checked how to do it properly.

**Comment by Kampi** [comment]
Gogo merge :D

**Comment by jakkra** [comment]
> Gogo merge :D

There are some minot conflicts, can you fix them?

**Comment by Kampi** [comment]
> > Gogo merge :D
> 
> There are some minot conflicts, can you fix them?

Yeah I can try it later.

**Pull request merged**